packages/backend/src/modules/dashboard/dashboard.controller.ts

```diff
@@ -0,0 +1,27 @@
+import CreateDashboardCommand from 'packages/shared/src/api/dto/CreateDashboardCommand.dto'
+import express from 'express'
+import dashboardService from './dashboard.service'
+
+class DashboardController {
+  // eslint-disable-next-line class-methods-use-this
+  async createDashboard(req: express.Request, res: express.Response) {
+    console.log(`Received new dashboard with title: ${req.body.title}`)
+    const createDashboardCommand: CreateDashboardCommand = req.body
+    const Dashboard = await dashboardService.createDashboard(createDashboardCommand)
+    return res.status(200).send(Dashboard.id)
```

Comment:
use 201 status for create, and return created daschboard with id generated in mongo